turtlebot3_node:
  ros__parameters:
#今日の天気
    opencr:
      id: 200
      baud_rate: 1000000
      protocol_version: 2.0

    wheels:
      separation: 0.160
      radius: 0.033

    motors:
      profile_acceleration_constant: 214.577

      # [rev/min2]
      # ref) http://emanual.robotis.com/docs/en/dxl/x/xl430-w250/#profile-acceleration
      profile_acceleration: 0.0

    sensors:
      bumper_1: 0
      bumper_2: 0
      illumination: 0
      ir: 0
      sonar: 0

diff_drive_controller:
  ros__parameters:

    odometry:
      publish_tf: true
      use_imu: true
      frame_id: "odom"
      child_frame_id: "base_footprint"
